lib/theming/src/types.ts

```diff
@@ -27,6 +27,8 @@ export interface ThemeVars {
   barTextColor?: string;
   barSelectedColor?: string;
   barBg?: string;
+  barIconHoverColor?: string;
+  barIconActiveColor?: string;
```

Comment:
Can you show me an example on why the icon should be different in color from the text?

Reply:
Personally I don't have use-case for this but I there was scenario raised in the issue where this would be useful: 

> Currently, the ThemeVars interface accepts a colorSecondary parameter which is used to color the selected item on the left sidebar and the toolbar icons hover state. However, in my custom theme I need the toolbar icon hover color to be something diffrerent because I'm using the same color for barBg and colorSecondary.

https://github.com/storybookjs/storybook/issues/13578 

It seemed like a reasonable feature to me but please let me know if I don't think it's worth the extra complexity and can close this PR.

Reply:
Yeah I personally don't think adding this to our ThemeVars is worth the extra complexity indeed.
We keep the ThemeVars object small on purpose, we know this means theming is limited.

Reply:
No worries! Have a great day! 🤔